### ISSUE TITLE
Offline Distro does not start

### ISSUE THREAD

**Issue by kaikreuzer** [openhab/openhab-distro#7]
I just tried the offline distro on a RasPi, which has no local Maven repo and thus is a clean environment.
When starting the runtime, I have these errors in the openhab.log:
```
Jan 03, 2016 5:19:25 PM org.apache.karaf.main.Main$KarafLockCallback lockAquired
INFO: Lock acquired. Setting startlevel to 100
2016-01-03 17:19:49.010 [WARN ] [url.mvn.internal.AetherBasedResolver] - Error resolving artifactorg.apache.karaf.features:framework:xml:features:4.0.3:Could not find artifact org.apache.karaf.features:framework:xml:features:4.0.3
shaded.org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact org.apache.karaf.features:framework:xml:features:4.0.3
	at shaded.org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve(DefaultArtifactResolver.java:444)[4:org.ops4j.pax.url.mvn:2.4.3]
...
2016-01-03 17:19:49.330 [ERROR] [ternal.service.BootFeaturesInstaller] - Error installing boot feature repository mvn:org.apache.karaf.features/standard/4.0.3/xml/features
java.io.IOException: Error resolving artifact org.apache.karaf.features:standard:xml:features:4.0.3: Could not find artifact org.apache.karaf.features:standard:xml:features:4.0.3 : mvn:org.apache.karaf.features/standard/4.0.3/xml/features
	at org.apache.karaf.features.internal.service.RepositoryImpl.load(RepositoryImpl.java:85)[9:org.apache.karaf.features.core:4.0.3]
	at org.apache.karaf.features.internal.service.FeaturesServiceImpl.loadRepository(FeaturesServiceImpl.java:391)[9:org.apache.karaf.features.core:4.0.3]
...
2016-01-03 17:19:49.544 [ERROR] [ternal.service.BootFeaturesInstaller] - Error installing boot feature repository mvn:org.openhab.distro/openhab-aggregate-xml/2.0.0-SNAPSHOT/xml/features
java.io.IOException: Error resolving artifact org.openhab.distro:openhab-aggregate-xml:xml:features:2.0.0-SNAPSHOT: Could not find artifact org.openhab.distro:openhab-aggregate-xml:xml:features:2.0.0-SNAPSHOT : mvn:org.openhab.distro/open
hab-aggregate-xml/2.0.0-SNAPSHOT/xml/features
	at org.apache.karaf.features.internal.service.RepositoryImpl.load(RepositoryImpl.java:85)[9:org.apache.karaf.features.core:4.0.3]
```
@dvanherbergen & @maggu2810, Did we miss to package something?

**Comment by maggu2810** [comment]
Sorry @kaikreuzer I stopped testing the oh distro as @dvanherbergen started that work. Just because a matter of time.
What is the content of the system directory?

**Comment by maggu2810** [comment]
@dvanherbergen Just to state: You have done a great work on the Karaf migration. Thank you.

**Comment by dvanherbergen** [comment]
@maggu2810 Thanks.
@kaikreuzer I'll have a look at is.

**Comment by kaikreuzer** [comment]
FTR: The online distro does not have this problem, but it fails when installing the demo feature with things like:
```
2016-01-03 18:15:03.335 [WARN ] [url.mvn.internal.AetherBasedResolver] - Error resolving artifactde.maggu2810.playground.com.eclipsesource.jaxrs:swagger-all:jar:1.5.4.sp2:Could not find artifact de.maggu2810.playground.com.eclipsesource.jaxrs:swagger-all:jar:1.5.4.sp2 in oh-releases-repo (https://jcenter.bintray.com/)
shaded.org.eclipse.aether.resolution.ArtifactResolutionException: Could not find artifact de.maggu2810.playground.com.eclipsesource.jaxrs:swagger-all:jar:1.5.4.sp2 in oh-releases-repo (https://jcenter.bintray.com/)
```
Thanks @dvanherbergen for looking at it!

**Comment by maggu2810** [comment]
So, the online repo does perhaps miss the repo. But as already stated, there is a new shk and the swagger modified artifact etc. on a bintray repo that should be used instead of the github one. So, the online dist. should use the bintray one.

**Comment by maggu2810** [comment]
The repo in system is not used anymore.

```bash
cat /tmp/strace.log  | grep openhab-aggregate-xml
```

```text
16282 stat("/home/maggu2810/.m2/repository/org/openhab/distro/openhab-aggregate-xml/2.0.0-SNAPSHOT/maven-metadata-local.xml", 0x7f110e8817a0) = -1 ENOENT (No such file or directory)
16282 stat("/home/maggu2810/.m2/repository/org/openhab/distro/openhab-aggregate-xml/2.0.0-SNAPSHOT/openhab-aggregate-xml-2.0.0-SNAPSHOT-features.xml", 0x7f110e881ab0) = -1 ENOENT (No such file or directory)
```

**Comment by maggu2810** [comment]
org.ops4j.pax.url.mvn.cfg miss the lines:
```text
org.ops4j.pax.url.mvn.defaultRepositories=\
    file:${karaf.home}/${karaf.default.repository}@id=system.repository@snapshots, \
    file:${karaf.data}/kar@id=kar.repository@multi@snapshots, \
    file:${karaf.base}/${karaf.default.repository}@id=child.system.repository@snapshots
```

**Comment by maggu2810** [comment]
I will create a PR after testing...

**Comment by maggu2810** [comment]
* offline dist: solved by #9
* online dist: we should create a separate issue for that
